auto_tag/detectors.py

```
@@ -55,13 +52,13 @@ def change_type(self) -> int:
 
-    def validate_detector_params(self) -> NoReturn: # type: ignore
+    def validate_detector_params(self) -> NoReturn:  # type: ignore
         """Check if all the parameters given to the detector make sens."""
         if self._change_type_name not in constants.CHANGE_TYPES.values():
             raise exception.DetectorValidationException(
-                ('Change type {} is not in not valid. Accepted'
-                 ' change types are {}').format(
-                     self._change_type_name, constants.CHANGE_TYPES.keys()))
+                f"Change type {self._change_type_name} is not in not valid. Accepted"
+                f" change types are {constants.CHANGE_TYPES.keys()}"
+            )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)